app/models/profile.rb

```diff
@@ -44,10 +44,15 @@ class Profile < ApplicationRecord
 
   # Profile has attached logo.
   has_attached_file :logo,
-    styles: { large: '300x300>'.freeze, medium: '150x150>'.freeze, thumb: '100x100>'.freeze, invoice: ['200x50>'.freeze, :png].freeze }.freeze,
-    :url => ':url_root/:account_id/:class/:attachment/:style/:basename.:extension'.freeze,
-    :s3_permissions => 'public-read'.freeze,
-    :default_url => '/assets/3scale-logo.png'.freeze
+    styles: {
+      large: '300x300>'.freeze,
+      medium: '150x150>'.freeze,
+      thumb: '100x100>'.freeze,
+      invoice: ['200x50>'.freeze, :png].freeze }.freeze,
+    url: ':url_root/:account_id/:class/:attachment/:style/:basename.:extension'.freeze,
+    s3_permissions: 'public-read'.freeze,
+    default_url: '/assets/3scale-logo.png'.freeze,
+    adapter_options: { hash_digest: Digest::SHA256 }
```

Comment:
For this we need to rehash all attachments which might be an issue for updating existing deployments. See [paperclip docs](https://www.rubydoc.info/gems/paperclip/6.1.0#checksum-fingerprint).

Reply:
```ruby
CLASS=Profile ATTACHMENT=logo rake paperclip:refresh:fingerprints
```

Reply:
we are not fingerprinting in fact